### PR TITLE
Add CTRL + ALT + O as open document shortcut.

### PR DESCRIPTION
**Pull request by gokaysatir**
* This needs to be handled by the integrator.

Change-Id: If57bdd1f7db496766fbb7082ffeff855aba4e97c

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: browser/src/map/handler/Map.Keyboard.js [hunk 1]
```diff
@@ -741,6 +741,13 @@ L.Map.Keyboard = L.Handler.extend({
 			return true;
 		}
 
+		// CTRL + ALT + O for new document. This needs to be handled by the integrator.
+		if (this._isCtrlKey(e) && e.altKey && e.key.toUpperCase() === 'O') {
+			this._map.fire('postMessage', { msgId: 'OpenDocument' });
```
Comment (eszkadev) [inline browser/src/map/handler/Map.Keyboard.js:746]: It doesn't use standard convention we have for other commands:    COOL -> Host    UI_xxxx      (but there are already 2 exceptions... so not sure :) )

We have already command to create new document: `UI_CreateFile`
How this is different?
https://sdk.collaboraonline.com/docs/postmessage_api.html#id6

Comment (gokaysatir) [reply browser/src/map/handler/Map.Keyboard.js]: Changed the command name to "UI_OpenDocument" in order to follow the rules, thanks :)

File: browser/src/map/handler/Map.Keyboard.js [hunk 2]
```diff
@@ -741,6 +741,13 @@ L.Map.Keyboard = L.Handler.extend({
 			return true;
 		}
 
+		// CTRL + ALT + O for new document. This needs to be handled by the integrator.
```
Comment (eszkadev) [inline browser/src/map/handler/Map.Keyboard.js:744]: copy paste from ctrl + alt + n? 
not for new document, but rather for opening other document?

**Comment by eszkadev** [comment]
please prepare PR for SDK with new command

**Pull request merged**